docker/scripts/osie.sh

```diff
@@ -425,7 +425,7 @@ EOF_tty
 	# Backup phone-home
 
 	# Add phone-home to OS
-	nonsystemdoses=(ubuntu_14_04 ubuntu_14_04_image scientific_6)
+	nonsystemdoses=(_14_04 ubuntu_14_04_image scientific_6)
```

Comment:
To my untrained eye, I'm not clear on why this change was made - can you explain? Doesn't need a comment, I'm just looking to learn.

Reply:
This was unintentional, great catch